index.html

```diff
@@ -36,5 +36,151 @@ <h5 class="connect">LET'S CONNECT</h5>
         </section>
       </main>
     </div>
+
+    <div>
+      <section>
+        <div class="garden">
+
+          <div class="water1">
+          <div class="portfolio"><img src="./img/Snapshoot-Portfolio.png" alt="portfolio"></div>
+          <h2 class="tonic">Tonic</h2>
+          <ul class="info">
+              <li class="tonic-detail">CANOPY</li>
+              <li><img src="./img/Counter.png" alt="punto"></li>
+              <li class="tonic-detail2">Back End Dev</li>
+              <li><img src="./img/Counter.png" alt="counter"></li>
+              <li class="tonic-detail2">2015</li>
+          </ul>
+          <p class="project-detail">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <div>
+              <ul class="buttons">
+                  <li><button class="int-button">html</button></li>
+                  <li><button class="int-button">css</button></li>
+                  <li><button class="int-button">JavaScript</button></li>
+              </ul>
+          </div>
+          <div><button class="see">See Project</button>
+          </div>
+       </div>
+
+
+          <div class="water2">
+            <div class="project2"><img src="./img/Snapshoot-project2.png" alt="project2"></div>
+            <h2 class="multi-post">Multi-Post Stories</h2>
+            <ul class="info">
+              <li class="tonic-detail">CANOPY</li>
+              <li><img src="./img/Counter.png" alt="punto"></li>
+              <li class="tonic-detail2">Back End Dev</li>
+              <li><img src="./img/Counter.png" alt="counter"></li>
+              <li class="tonic-detail2">2015</li>
+            </ul>
+            <p class="project-detail">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div>
+                <ul class="buttons">
+                    <li><button class="int-button">html</button></li>
+                    <li><button class="int-button">css</button></li>
```

Comment:
- [ ] Kindly try to stick to the figma design as much as possible to make your work more professional and presentable. you can do so by matching font colors,padding,font-size,margins,etc.

Figma design:
![image](https://user-images.githubusercontent.com/90524466/185734039-c7f8158c-fa3a-42a9-9a9a-e7db43a635cd.png)

Your design:
![image](https://user-images.githubusercontent.com/90524466/185734058-09c12a66-e967-41b0-8e69-566ad3d8e956.png)